app/src/main/res/layout/fragment_game.xml

```diff
@@ -1,6 +1,127 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/purple_500">
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/player_turn_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="18dp"
+            android:textAlignment="center"
+            android:textColor="#8C9EFF"
```

Comment:
it's okay to hard code colors, but one thing to potentially be thinking about in the future is how you might support night mode for your application. nothing we really need to worry about with this project, as we can tackle themes as a whole  on another project, but just something to be aware of!